CMakeLists.txt

```diff
@@ -17,29 +17,17 @@ if (NOT SKBUILD)
   in your environment once and use the following command that avoids
   a costly creation of a new virtual environment at every compilation:
   =====================================================================
-   $ python setup.py install
+   $ pip install --no-build-isolation -v .
   =====================================================================")
 endif()
 
-# Perform a release build by default
-if (NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
-  set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
-  set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
-endif()
-
 # Create CMake targets for all Python components needed by nanobind
-if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.26 AND Python_INTERPRETER_ID STREQUAL "Python")
+if(Python_INTERPRETER_ID STREQUAL "Python")
```

Comment:
What is this? Looks like an output, I know I don't set it as an input.

For now, I'd recommend always adding the SABI component. There will be an input variable to check soon.